modules/EmailMan/EmailManDelivery.php

```diff
@@ -60,6 +60,9 @@
 $mail = new SugarPHPMailer();
 $admin = new Administration();
 $admin->retrieveSettings();
+if (isset($admin->settings['mail_sendtype']) && !empty($admin-settings['mail_sendtype']) && $admin->settings['mail_sendtype'] == "SMTP") {
```

Comment:
`$admin->settings['mail_sendtype']` instead of `$admin-settings['mail_sendtype']`